pyproject.toml

```diff
@@ -24,7 +24,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
+python = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4"
```

Comment:
This can probably use same requirements as circulation: 
https://github.com/ThePalaceProject/circulation/blob/main/pyproject.toml#L60